deployments/nfd/overlays/node-feature-rules/node-feature-rules-openshift.yaml

```diff
@@ -0,0 +1,100 @@
+# this is a copy of node-feature-rules.yaml but it is tailored
+# for openshift platforms.
+apiVersion: nfd.openshift.io/v1
+kind: NodeFeatureDiscovery

```

Comment:
@mregmi there's a small misunderstanding. I wanted to keep `NodeFeatureDiscovery` and `NodeFeatureRule` definitions in their own files.

Reply:
Fixed